tools/kotlin-native-gradle-plugin/src/main/kotlin/org/jetbrains/kotlin/gradle/plugin/konan/KonanPlugin.kt

```diff
@@ -363,15 +363,36 @@ class KonanPlugin @Inject constructor(private val registry: ToolingModelBuilderR
             doLast { project.cleanKonan() }
         }
 
+        project.afterEvaluate {
+            project.tasks
+                .withType(KonanCompileProgramTask::class.java)
+                .forEach { task ->
```

Comment:
Doesn't this call create all the tasks of the `KonanCompileProgramTask` class? 
Do we use the run tasks somehow? May be just drop them? :)

Reply:
Yes, this call will create tasks. As the task creation requires properties from the build task, this could not be avoided. It seems that we don't use these run tasks so it should not be an issue. I think it's ok to leave them

Reply:
> It seems that we don't use these run tasks so it should not be an issue.

But we still execute this `afterEvaluate { ... }` on each configuration thus create all the `KonanCompileProgramTask`s, don't we?